code/controller/TranslatableCMSTreeNodeExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @package translatable
+ */
+class TranslatableCMSTreeNodeExtension extends Extension {
+
+	function init() {
+		die("Init TranslatableCMSTreeNodeExtension. "); // this never gets called in the current config
+	}
+
+	/**
+	 * Overrides {@link LeftAndMain_TreeNode->forTemplate()} in order to show the original page name on mouseover.
+	 *
+	 * @todo Make it more flexible, because right now it's a simple copy-paste with changes
+	 *
+	 * @return String
+	 */
+	public function forTemplate() {
```

Comment:
Couldn't this use a template? Maybe the real fix is to make core use a template here then we can override it with translatable?